scala-test-suite/src/test/resources/2.11.8/WhitelistedTests.txt

```diff
@@ -2,6 +2,7 @@ scala/collection/IndexedSeqOptimizedTest.scala
 scala/collection/IterableViewLikeTest.scala
 scala/collection/SearchingTest.scala
 scala/collection/TraversableOnceTest.scala
+scala/collection/immutable/PagedSeqTest.scala
```

Comment:
Added this test to white list because I did not find a configuration in which it failed.